internal/cocoa/api_cocoa_darwin.go

```diff
@@ -38,23 +43,165 @@ var (
 	sel_setArgumentAtIndex                 = objc.RegisterName("setArgument:atIndex:")
 	sel_getReturnValue                     = objc.RegisterName("getReturnValue:")
 	sel_invoke                             = objc.RegisterName("invoke")
+	sel_invokeWithTarget                   = objc.RegisterName("invokeWithTarget:")
 	sel_instanceMethodSignatureForSelector = objc.RegisterName("instanceMethodSignatureForSelector:")
 	sel_signatureWithObjCTypes             = objc.RegisterName("signatureWithObjCTypes:")
 	sel_initWithUTF8String                 = objc.RegisterName("initWithUTF8String:")
 	sel_UTF8String                         = objc.RegisterName("UTF8String")
 	sel_length                             = objc.RegisterName("length")
+	sel_processInfo                        = objc.RegisterName("processInfo")
+	sel_isOperatingSystemAtLeastVersion    = objc.RegisterName("isOperatingSystemAtLeastVersion:")
+	sel_frame                              = objc.RegisterName("frame")
+	sel_size                               = objc.RegisterName("size")
+	sel_contentView                        = objc.RegisterName("contentView")
+	sel_setBackgroundColor                 = objc.RegisterName("setBackgroundColor:")
+	sel_colorWithSRGBRedGreenBlueAlpha     = objc.RegisterName("colorWithSRGBRed:green:blue:alpha:")
+	sel_setFrameSize                       = objc.RegisterName("setFrameSize:")
+	sel_object                             = objc.RegisterName("object")
+	sel_styleMask                          = objc.RegisterName("styleMask")
+	sel_setStyleMask                       = objc.RegisterName("setStyleMask:")
+	sel_mainScreen                         = objc.RegisterName("mainScreen")
+	sel_screen                             = objc.RegisterName("screen")
+	sel_isVisible                          = objc.RegisterName("isVisible")
+	sel_deviceDescription                  = objc.RegisterName("deviceDescription")
+	sel_objectForKey                       = objc.RegisterName("objectForKey:")
+	sel_unsignedIntValue                   = objc.RegisterName("unsignedIntValue")
 )
 
-type CGFloat float64
+const NSWindowCollectionBehaviorFullScreenPrimary = 1 << 7
+const (
+	NSWindowStyleMaskResizable  = 1 << 3
+	NSWindowStyleMaskFullScreen = 1 << 14
+)
+
+type CGFloat = float64
 
 type CGSize struct {
 	Width, Height CGFloat
 }
 
+type CGPoint struct {
+	X, Y float64
+}
+
+type CGRect struct {
+	Origin CGPoint
+	Size   CGSize
+}
+
+type NSUInteger = uint
+type NSInteger = int
```

Comment:
I slightly prefer making them different types

```go
type NSUInteger uint
type NSInteger int
```

What do you think?

Reply:
It was frustrating to forget the cast so doing a type alias was easier. Also, [Apple](https://developer.apple.com/documentation/objectivec/nsuinteger) describes it as a type alias.

Reply:
I understood.